docs/extending/samples/clusterprovider/contosocloud.ts

```diff
@@ -0,0 +1,139 @@
+/*
```

Comment:
is this entire file meant to be commented?

Reply:
Yes... if I don't comment it, then it causes build warnings on the extension which is undesirable (and no, excluding the directory doesn't seem to help).  I think I need to move samples out to a separate samples repo which can depend on the NPM package - thoughts?